.github/workflows/basic-eval.yml

```diff
@@ -0,0 +1,20 @@
+name: Basic evaluation checks
+
+on:
+  pull_request:
+    branches:
+     - master
+     - release-**
+  push:
+    branches:
+     - master
+     - release-**
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    # we don't limit this action to only NixOS repo since the checks are cheap and useful developer feedback
```

Comment:
I don't agree with this one. Normally when the tarball does not build on hydra the error messages are always very cryptic or do not display any helpful information at all. So far I was not able to fix any of those build failures on hydra when they blocked a channel advancement and I think my nix knowledge is above average and I should be able to fix that or at least diagnose the problem.

Reply:
That's an orthogonal problem? This job only gives you feedback that it will block the channel if merged. So the issue has to be addressed before the PR goes in.

Reply:
Hmmm.. Maybe we should just try it and see how it goes.